dev/ci/ci-wrapper.sh

```diff
@@ -46,6 +46,10 @@ else
   bash "${DIR}/${CI_SCRIPT}" 2>&1 | tee "$CI_NAME.log"
 fi
 code=$?
+
+echo "$code" > "$CI_NAME.exitcode"
+echo "exitcode: $code" >> "$CI_NAME.log"
```

Comment:
The .log files are gitignored but I don't think we should produce more files.
If you do
```suggestion
echo "$CI_NAME exit code: $code" >> "$CI_NAME.log"
```
you can still get build statuses by doing `tail -q -n 1 *.log` (followed by some grep if you want)

Reply:
What you suggest works if we can relyi on CI makefiles to never write to their stdout (the log file) after the toplevel process exits. This may be true, but I don't think there's any way we would know if it wasn't.

Reply:
Since the standard output of the jobs is `tee`-ed into the log files, you would need the jobs to be actively adversarial (i.e., demonizing a background process and then manually reopening the log file) for them to be able to write something to the log files after the exit code.

Reply:
Okay, I am applying this suggestion. I did add an extra newline first, as some logs I saw did not have one before the exit-code line.